contracts/solidity/precompiles/Precompiles.sol

```diff
@@ -5,6 +5,7 @@ contract Precompiles {
 
     event DebugBytes(bytes data);
     event DebugUint256(uint256 value);
+    uint256 constant dummy = 0;
```

Comment:
Q: where's `dummy` used?

Reply:
Right I was adding it attempting to remove a warning within the smart contract but didn't need it somehow and ended up forgetting about it. Great catch! Pushed in the fix!